renovate.json

```diff
@@ -75,6 +75,17 @@
       "matchPackageNames": [
         "*"
       ]
-    }
+    },
+    {
+      "customType": "regex",
+      "fileMatch": [
+        "/Chart.yaml$"
+      ],
+      "matchStrings": [
+        "\\s*#\\s?renovate: (?<datasource>.*?)=(?<depName>.*?)( versioning=(?<versioning>.*?))?\\s+[\\w+\\.\\-]+(?:[:=]|\\s+\\S+)\\s*[\\\"']?(?<currentValue>[\\w+\\.\\-]*)(?:@(?<currentDigest>sha256:[a-f0-9]+))?[\\\"']?"
+      ],
+      "datasourceTemplate": "{{#if (equals datasource 'github')}}github-tags{{else}}{{{datasource}}}{{/if}}",
+      "versioningTemplate": "{{#if (equals datasource 'docker')}}docker{{else if versioning}}{{{versioning}}}{{else}}semver{{/if}}"
+    },
```

Comment:
this section is invalid in context of `packageRules`. A customManager that handles the comments is already present around line 15.

Reply:
I have removed the redundant section